content/en/docs/tasks/tools/included/optional-kubectl-configs-fish.md

```diff
@@ -13,3 +13,5 @@ kubectl completion fish | source
 ```
 
 After reloading your shell, kubectl autocompletion should be working.
+
+Note: this requires kubectl 1.23 or later.
```

Comment:
```suggestion
{{< note >}}
Autocomplete for Fish requires kubectl 1.23 or later.
{{< /note >}}
```

Reply:
Hi @themarquee . As suggested, I think it makes sense to use the `note` shortcode. What do you think?

Reply:
Hello @themarquee , please could you let us know if this is something you'd be able to work upon?